src/Application/ActivityFeed/Queries/GetActivitiesQuery.cs

```diff
@@ -32,19 +32,19 @@ public async Task<ActivityFeedViewModel> Handle(GetActivitiesQuery request, Canc
         
         var staffDtos = await _dbContext.StaffMembers
             .Join(_dbContext.Users,
-                staff => staff.Email, 
-                user => user.Email,
-                (staff, user) =>
+                s => s.Email, 
```

Comment:
Not critical, but `staff` and `user` seem to be easier to read than `s` and `u`. Can you please keep them?

Reply:
Done - reverted this